### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by yoong93** [RustPython/RustPython#2745] [Q&A] [answered]
Hi, I'm interested in participating, trying to build the project before that. I have installed in my WSL(Ubuntu), installed rust 1.53, and cloned the project. But when I try building the project, I keep getting error message.

```
error[E0599]: no function or associated item named `try_from_object` found for struct `CString` in the current scope
    --> vm/src/stdlib/os.rs:1700:54
     |
1700 |                     let cstring = std::ffi::CString::try_from_object(vm, obj)?;
     |                                                      ^^^^^^^^^^^^^^^ function or associated item not found in `CString`
     |
     = help: items from traits can only be used if the trait is implemented and in scope
note: `pyobject::TryFromObject` defines an item `try_from_object`, perhaps you need to implement it``
```
I'm relatively new to rust and couldn't find direct solution regards to error, couldn't find try_from_object from CString docs.
Any help on this? Thanks in advance.

**Comment by DimitrisJim** [comment, marked as answer]
An error indirectly got through on the master branch, see #2743. You could either `git checkout HEAD~1` and play with that for now or wait until it a fix is merged :-)

**Reply by yoong93** [reply]
Worked out nicely! Thanks :)
